pdm.lock

```diff
@@ -3,7 +3,7 @@
 
 [metadata]
 groups = ["default", "all", "cookiecutter", "copier", "doc", "keyring", "pytest", "template", "test", "tox", "workflow"]
-strategy = ["inherit_metadata"]
+strategy = []
```

Comment:
Do you turn off the `strategy.inherit_metadata` config?

Please do not remove this strategy

Reply:
ok!

Reply:
no its True....
(.venv) ➜  pdm git:(main) pdm config | grep strategy                  
strategy.inherit_metadata = True
strategy.resolve_max_rounds = 10000
strategy.save = minimum
strategy.update = reuse

Reply:
⠋ 0:00:00 Resolving dependencies  WARNING: inherit_metadata strategy is not supported by uv resolver, it will be ignored

I know the reason

Reply:
ok, uv resolver is not 100% compatible, use pdm for project lock file instead.

Reply:
also found it, and resolve